frontend/src/metabase/search/components/InfoText.tsx

```diff
@@ -26,7 +31,7 @@ const infoTextPropTypes = {
   result: PropTypes.shape(searchResultPropTypes),
 };
 
-export function InfoText({ result }) {
+function getInfoText(result: WrappedResult) {
```

Comment:
good ole typescript pointed out since this only returned strings or arrays it wasn't even a valid React component.